File: LinkedList/1D LinkedList/traversal/traversal.ts
class ListNode {
    val: number;
    next: ListNode | null;

    constructor(val: number, next: ListNode | null = null) {
        this.val = val;
        this.next = next;
    }
}

class Solution {
    getLength(head: ListNode | null): number {
        let count = 0;
        let temp = head;

        while (temp !== null) {
            count++;
            temp = temp.next;
        }

        return count;
    }
}

File: LinkedList/traversal/traversal.py
# Definiton of singly Linked List
# class ListNode:
#     def __init__(self, val=0, next=None):
#         self.val = val
#         self.next = next

class Solution:
    def insertAtHead(self, head, X):
        temp = ListNode(X)
        temp.next = head
        return temp

        